docs/html-css/week-1/lesson.md

```diff
@@ -22,15 +25,31 @@ Learning objectives for this lesson can be found [here](./learning-objectives.md
 
 ## What makes a web page?
 
-<img src={require('!file-loader!../assets/webpage-parts.png').default}/>
+<img src={require('!file-loader!../assets/webpage-parts.png').default}
+  alt="Diagram showing that structure (HTML), presentation (CSS), and behaviour (JavaScript) are interlinked"/>
 
-**Separation of Concerns:** In computer science, separation of concerns (SoC) is a design principle for separating a computer program into distinct sections, such that each section addresses a separate concern. (Wikipedia)
+A web page consists of three programming languages: HTML, CSS, and JavaScript.
+They define the structure (HTML), presentation (CSS), and behaviour (JavaScript) of the page.
```

Comment:
The image is briefly explained here so that when students refer back to the lesson they don't have to guess what the image is saying and have a concrete explanation.